mandatory/4-tax.js

```diff
@@ -17,7 +23,12 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let finalPrice = (price *1.2).toFixed(2)
```

Comment:
On the part `(price * 1.2)`, you can also use the `calculateSalesTax()` function you just wrote to make things a bit simpler.